Update README.md

Katjadoerle opened the pull request:
Bitte bei den Änderungen in Abschnitt 4. nochmal die Aussage kontrollieren.

Katjadoerle committed:
Update README.md

das-g commented on README.md:
```diff
@@ -11,39 +11,39 @@ Hier findest du die technischen Präsentationen des Workshops:
 ### 1. Die Kommandozeile \(Samstag Morgen\)
 
-Die nächsten zwei Tage werden wir ganz viel mit der Kommandozeile arbeiten. Gehe durch das Kapitel ["Einführung in die Kommandozeile"](https://tutorial.djangogirls.org/de/intro_to_command_line/) um dich damit vertraut zu machen und die Basics zu lernen.
+Die nächsten zwei Tage werden wir häufig mit der Kommandozeile arbeiten. Gehe durch das Kapitel ["Einführung in die Kommandozeile"](https://tutorial.djangogirls.org/de/intro_to_command_line/), um dich damit vertraut zu machen und die Basics zu lernen.
```
"ganz viel" würde ich lassen, denn
- die Kommandozeile wird im Tutorial (anteilsmässig) viel mehr verwendet, als die typische Teilnehmerin (=nicht IT-lerin) das normalerweise bei der Computer-Benutzung bisher getan haben wird
- das Tutorial ist absichtlich in einer leicht Umgangssprachlichen Form gehalten, um gefälliger zu wirken

das-g replied:
Mit f4c011a415025730a0051295b4cfc046b096ea18 auf "ganz viel" zurückgestellt

das-g commented on README.md:
```diff
@@ -11,39 +11,39 @@ Hier findest du die technischen Präsentationen des Workshops:
 ### 3. Einführung in Python \(Samstag Nachmittag\)
 
-Jetzt wird es richtig nerdig \(aber keine Sorge, dein Coach wird dir helfen\), wir lassen unsere statische Website ruhen und folgen dem Kapitel [Introduction to python](https://tutorial.djangogirls.org/en/python_introduction/) aus dem Django Girls Tutorial. Wir werden Python morgen brauchen um die nächste Herausforderung zu bewältigen ne dynamische Webseite zu erstellen.
+Es wird richtig nerdig \(aber keine Sorge, dein Coach wird dir helfen\)! Wir lassen unsere statische Webseite ruhen und folgen dem Kapitel [Introduction to python](https://tutorial.djangogirls.org/en/python_introduction/) aus dem Django Girls Tutorial. Wir werden Python morgen brauchen, um die nächste Herausforderung zu bewältigen: eine dynamische Webseite erstellen.
```
Ich würde eine pseudo-Zeitangabe im Abschnitt-Anfang lassen. Falls man nicht zwei aufeinanderfolgende Abschnitte mit "Jetzt wird/wirst" beginnen will, vielleicht "Nun [...]" für einen von beiden?

das-g replied:
Mit ed2e383c551b253c7eabe3ce6876f56730c536cf auf "nun" umgestellt

das-g commented on README.md:
```diff
@@ -11,39 +11,39 @@ Hier findest du die technischen Präsentationen des Workshops:
 ### 3. Einführung in Python \(Samstag Nachmittag\)
 
-Jetzt wird es richtig nerdig \(aber keine Sorge, dein Coach wird dir helfen\), wir lassen unsere statische Website ruhen und folgen dem Kapitel [Introduction to python](https://tutorial.djangogirls.org/en/python_introduction/) aus dem Django Girls Tutorial. Wir werden Python morgen brauchen um die nächste Herausforderung zu bewältigen ne dynamische Webseite zu erstellen.
+Es wird richtig nerdig \(aber keine Sorge, dein Coach wird dir helfen\)! Wir lassen unsere statische Webseite ruhen und folgen dem Kapitel [Introduction to python](https://tutorial.djangogirls.org/en/python_introduction/) aus dem Django Girls Tutorial. Wir werden Python morgen brauchen, um die nächste Herausforderung zu bewältigen: eine dynamische Webseite erstellen.
```
Wenn wir den Satz vor "Wir lassen unsere ..." trennen (was IMHO korrekt ist :+1:), dann lass uns das doch gleich vor der Klammer machen. Dann wird deren Inhalt auch zu einem ganzen Satz. Etwa so:

> [...] richtig nerdig<strong>!</strong> \(**A**ber keine Sorge<strong>:</strong> **D**ein Coach wird dir helfen<strong>.</strong>\) **W**ir lassen unsere statische Webseite ruhen und [...]

(**Fett** nur zur Hervorhebung der Änderungen, soll nicht Teil der Änderung selbst werden.)

das-g replied:
gemacht mit 8c7993a83f6e645cafd60f584c35e162507e047c

das-g commented on README.md:
```diff
@@ -11,39 +11,39 @@ Hier findest du die technischen Präsentationen des Workshops:
 ### 3. Einführung in Python \(Samstag Nachmittag\)
 
-Jetzt wird es richtig nerdig \(aber keine Sorge, dein Coach wird dir helfen\), wir lassen unsere statische Website ruhen und folgen dem Kapitel [Introduction to python](https://tutorial.djangogirls.org/en/python_introduction/) aus dem Django Girls Tutorial. Wir werden Python morgen brauchen um die nächste Herausforderung zu bewältigen ne dynamische Webseite zu erstellen.
+Es wird richtig nerdig \(aber keine Sorge, dein Coach wird dir helfen\)! Wir lassen unsere statische Webseite ruhen und folgen dem Kapitel [Introduction to python](https://tutorial.djangogirls.org/en/python_introduction/) aus dem Django Girls Tutorial. Wir werden Python morgen brauchen, um die nächste Herausforderung zu bewältigen: eine dynamische Webseite erstellen.
```
Ich würde nach "nächste Herausforderung zu bewältigen" ein Komma anstatt einen Doppelpunkt setzen und das "zu" in "zu erstellen" dann dort belassen. Deine Variante geht aber auch.

das-g commented on README.md:
```diff
@@ -11,39 +11,39 @@ Hier findest du die technischen Präsentationen des Workshops:
 ### 4. Django: Dein Blog ist eine dynamische Webseite \(Sonntag\)
 
-Jetzt tauchen wir richtig ins Django Girls Tutorial ein. Ab dem Kapitel [What is Django?](https://tutorial.djangogirls.org/en/django/) geht 's los. Dein Django Blog ist ein Projekt unabhängig von deiner statische Webseite. Es muss also in einem anderen Ordner gestartet werden. Auch in Github wird es an einem anderen Ort abgelegt als das was du bis jetzt gemacht hast. Github kan nur statische Seit ...
+Ab dem Kapitel [What is Django?](https://tutorial.djangogirls.org/en/django/) geht 's los: wir tauchen richtig ins Django Girls Tutorial ein. Dein Django Blog ist ein von deiner statischen Webseite unabhängiges Projekt. Es muss also in einem anderen Ordner gestartet werden. Auch in Github wird es an einem anderen Ort abgelegt als das, was du bis jetzt gemacht hast. Wir werden die Codes immer noch ...
```
Der Sinn ist gut erhalten geblieben. Danke! :+1:

das-g commented on README.md:
```diff
@@ -11,39 +11,39 @@ Hier findest du die technischen Präsentationen des Workshops:
 
 ### 4. Django: Dein Blog ist eine dynamische Webseite \(Sonntag\)
 
-Jetzt tauchen wir richtig ins Django Girls Tutorial ein. Ab dem Kapitel [What is Django?](https://tutorial.djangogirls.org/en/django/) geht 's los. Dein Django Blog ist ein Projekt unabhängig von deiner statische Webseite. Es muss also in einem anderen Ordner gestartet werden. Auch in Github wird es an einem anderen Ort abgelegt als das was du bis jetzt gemacht hast. Github kan nur statische Seit ...
```
"GitHub" ohne bestimmten Artikel verwenden. (Also "in GitHub" anstatt "im GitHub".)

Tendenziell würde ich sogar eher "auf GitHub" als "in GitHub" schreiben, da es sich um eine Internetplattform handelt. (So wie "in Twitter" und "in Facebook" nicht falsch sind, aber "auf Twitter" und "auf Facebook" eigentlich üblicher.)

das-g replied:
gemacht: 9d68cc81843aeddc370bf6d77520bd553e35ad5c

das-g commented on README.md:
```diff
@@ -11,39 +11,39 @@ Hier findest du die technischen Präsentationen des Workshops:
 
 ### 4. Django: Dein Blog ist eine dynamische Webseite \(Sonntag\)
 
-Jetzt tauchen wir richtig ins Django Girls Tutorial ein. Ab dem Kapitel [What is Django?](https://tutorial.djangogirls.org/en/django/) geht 's los. Dein Django Blog ist ein Projekt unabhängig von deiner statische Webseite. Es muss also in einem anderen Ordner gestartet werden. Auch in Github wird es an einem anderen Ort abgelegt als das was du bis jetzt gemacht hast. Github kan nur statische Seit ...
```
Den Satz "Das klingt kompliziert, und das ist es auch." würde ich so belassen. "Das klingt nicht nur kompliziert, sondern ist es auch." ist (m.M.n. unnötig) länger, sagt das gleiche aus, aber betont es (m.M.n. unnötig) stark. "... und das ist es auch" ist beileibe heavy genug. Es geht hier ja nicht darum, die Teilnehmerinnen abzuschrecken, sondern sie schon im Voraus für das (vermutlich) künftig erreichte zu loben.

das-g replied:
Rückgängig gemacht: 62bff2e294dc299964dbdd5669466cc17da5ad9f

das-g commented on README.md:
```diff
@@ -11,39 +11,39 @@ Hier findest du die technischen Präsentationen des Workshops:
 ### C. Extras
 
-Je nach Zeit und Interesse kannst du während des Workshops, zu Hause oder an einem der Follow up Events an folgenden Sachen weiter arbeiten:
+Je nach Zeit und Interesse kannst du während des Workshops, zu Hause oder an einem der Follow up-Events an folgenden Sachen weiter arbeiten:
```
Wenn wir es (wie es das Deutsche erforder) Binden, dann ganz:

> Follow-Up-Event

oder mit "follow up" als Englischem Fremd-Terminus:

> "Follow up"-Event

(mit den Anführungszeichen!)

das-g replied:
gemacht: f9cf4f73ac5ea1564d09c544198bf6d0f360c9a6

das-g commented on README.md:
```diff
@@ -11,39 +11,39 @@ Hier findest du die technischen Präsentationen des Workshops:
 > Dieses Tutorial ist lizenziert unter der Creative Commons Attribution-ShareAlike 4.0 International Lizenz. Gehe auf [https://creativecommons.org/licenses/by-sa/4.0/](https://creativecommons.org/licenses/by-sa/4.0/) um eine Kopie der Lizenz zu sehen.
 >
-> Das Tutorial ist gemacht um in Zusammenspiel mit dem offiziellen Django Girls Tutorial bearbetiet zu werden und hat aus für die Zweckmässigkeit einige Teile davon dupliziert. \([https://tutorial.djangogirls.org/en](https://tutorial.djangogirls.org/de), lizenziert unter der Creative Commons Attribution-ShareAlike 4.0 International Lizenz\).
+> Das Tutorial ist gemacht, um im Zusammenspiel mit dem offiziellen Django Girls Tutorial bearbeitet zu werden und hat aus Zweckmässigkeit einige Teile davon dupliziert. \([https://tutorial.djangogirls.org/en](https://tutorial.djangogirls.org/de), lizenziert unter der Creative Commons Attribution-ShareAlike 4.0 International Lizenz\).
```
Den unbestimmten Artikel ("in Zusammenspiel" anstatt "im Zusammenspiel") hättest du belassen können. Aber mit dem Bestimmten ("im") ist es auch korrekt.

Deine anderen Korrekturen hier brauchts es natürlich. :+1:

das-g committed:
"ganz viel" würde ich lassen

denn
- die Kommandozeile wird im Tutorial (anteilsmässig) viel mehr verwendet, als die typische Teilnehmerin (=nicht IT-lerin) das normalerweise bei der Computer-Benutzung bisher getan haben wird
- das Tutorial ist absichtlich in einer leicht Umgangssprachlichen Form gehalten, um gefälliger zu wirken

das-g committed:
pseudo-Zeitangabe wieder eingebaut

allerdings jetzt "nun"

das-g committed:
Klammer aus Satz herausgenommen

das-g committed:
"GitHub" ohne bestimmten Artikel

und mit "auf" anstatt "in"

das-g committed:
Zurück zu weniger abschreckender Formulierung

siehe https://github.com/django-girls-switzerland/static-website-tutorial-german/pull/9#discussion_r209420438

das-g committed:
Bindestriche eingefügt
